.github/workflows/deploy.yml

```diff
@@ -28,19 +28,8 @@ jobs:
         fetch-depth: 0
         persist-credentials: false
 
-    - name: Set up Python
-      uses: actions/setup-python@v4
-      with:
-        python-version: "3.7"
-
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        pip install --upgrade build tox
-
-    - name: Build package
-      run: |
-        python -m build
+    - name: Build and Check Package
+      uses: hynek/build-and-inspect-python-package@v1.5
```

Comment:
Actually I'm not entirely sure that this will produce the packages in the standard location, so the `Publish package to PyPI` will be able to pick them up.

Looking at the action code, it builds the packages in a temporary directory: https://github.com/hynek/build-and-inspect-python-package/blob/0b1716730c334a22b313808557b14b6190896bc3/action.yml#L40

I will have to test this out in a separate repository before being confident this will work.

Reply:
FTR you can't access them from other actions (like the upload action) because it's under `/tmp`. Use a `download-artifact` like [here](https://github.com/python-attrs/attrs/blob/ff7f9de526eb586aa37bd800448ebb32976bf11e/.github/workflows/pypi-package.yml#L84-L88) (you may be also able to `cp -a` them I guess).

Reply:
Yeah, thanks, that's what I did here: https://github.com/pytest-dev/pytest-mock/pull/343

Reply:
Tested the same approach in https://github.com/pytest-dev/pytest-forked/pull/82 and it works well. 👍